Jenkinsfile

```diff
@@ -19,11 +18,25 @@ pipeline {
         stage('Fedora unit') {
           steps {
             sh readFile(file: 'jenkins/python3-tests.sh')
-            junit('coverage.xml')
-            // TODO: find the correct adapter or generate coverage tests that can be
-            //       parsed by an existing adapter:
-            //       https://plugins.jenkins.io/code-coverage-api/
-            // publishCoverage adapters: [jacocoAdapter('coverage.xml')]
+          }
+          post {
+            always {
+              archiveArtifacts allowEmptyArchive: true, artifacts: 'coverage.xml', fingerprint: true
+              // https://www.jenkins.io/doc/pipeline/steps/cobertura/
+              step([$class: 'CoberturaPublisher',
+                            autoUpdateHealth: false,
+                            autoUpdateStability: false,
+                            coberturaReportFile: 'coverage.xml',
+                            failNoReports: false,
+                            failUnhealthy: false,
+                            failUnstable: false,
+                            maxNumberOfBuilds: 10,
+                            onlyStable: false,
+                            sourceEncoding: 'ASCII',
+                            zoomCoverageChart: false])
```

Comment:
I picked a permissive set of options so it won't fail ever (we should decide what we want), but you get nice published reports now.  From the blue ocean click on "Go to Classic" (square with the right arrow) > Coverage report  .  The PR page in the classic view will also have  an overall graph.   This publisher also would make it worthwhile to start running the tests against the main branch once these PRs are merged so we could see longer term trends between PRs.